### ISSUE TITLE
Triggering submit outside of form should display validation messages

### ISSUE THREAD

**Issue by Devqon** [ngx-formly/ngx-formly#1845]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[x] support request
```

**Current behavior**
When trying to trigger a submit of the form from *outside* of the form, validation messages are not displayed.

**Expected behavior**
When triggering the submit of an `ngForm`, the validation should show. According to [this code](https://github.com/ngx-formly/ngx-formly/blob/177c4f45abbe46c3659e6e822ace8c471d8d6f07/src/core/src/lib/services/formly.config.ts#L35) this should be the default behavior.

**Minimal reproduction of the problem with instructions**
Put a button outside of the form, and trigger one/all of these:
- submit()
- myForm.ngSubmit.emit()
- formGroup.markAsTouched()
Validation is not shown (is the form not marked as submitted?).
See https://stackblitz.com/edit/ngx-formly-ui-bootstrap-nkj89o?file=app/app.component.ts

**What is the motivation / use case for changing the behavior?**
I want to show the validation messages when the user submits the form. The submit button however is outside of the `form` tags.

**Please tell us about your environment:**
Not applicable

* **Angular version:** 2.0.X
7.x

* **Browser:**
all

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
all

**Comment by aitboudad** [comment]
`ngSubmit.emit()`doesn't update the `submitted` property which we rely on,
for a workaround check https://stackblitz.com/edit/ngx-formly-ui-bootstrap-mm2ezi

**Comment by SudeshAK** [comment]
Hi @aitboudad with reference above workaround 
https://stackblitz.com/edit/ngx-formly-ui-bootstrap-l8gkcw 
here if you check console this.form.valid it will always return true though form is invalid can you please help me on this

**Comment by aitboudad** [comment]
@SudeshAK you've missed passing `form` input to the `formly-form` component:

```patch
<formly-form
+  [form]="form"
  [options]="options"
  [model]="model"
  [fields]="fields">
</formly-form>
```

**Comment by SudeshAK** [comment]
@aitboudad I have updated the example https://stackblitz.com/edit/ngx-formly-ui-bootstrap-l8gkcw
in this I have to validate a form only after it gets submitted outside of form for this I used expression properties but below if I put this.form.valid it always shows true

**Comment by aitboudad** [comment]
@SudeshAK I'm able to reproduce it, I'll let you know once fixed.

**Comment by aitboudad** [comment]
@SudeshAK fixed in [v5.5.11](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.5.11)

**Comment by SudeshAK** [comment]
Its working... thanks
